source/widgets/visualize/distributions.md

```diff
@@ -13,29 +13,42 @@ Displays value distributions for a single attribute.
 
-![](images/Distributions-Discrete.png)
+![](images/Distributions-Discrete-stamped.png)
 
 1. A list of variables for display. *Sort categories by frequency* orders displayed values by frequency.
-2. Set *Bin width* with the slider. Precision scale is set to sensible intervals. *Fitted distribution* fits selected distribution to the plot. Options are [Normal](https://en.wikipedia.org/wiki/Normal_distribution), [Beta](https://en.wikipedia.org/wiki/Beta_distribution), [Gamma](https://en.wikipedia.org/wiki/Gamma_distribution), [Rayleigh](https://en.wikipedia.org/wiki/Rayleigh_distribution), [Pareto](https://en.wikipedia.org/wiki/Pareto_distribution), [Exponential](https://en.wikipedia.org/wiki/Exponential_distribution), [Kernel density](https://en.wikipedia.org/wiki/Kernel_density_estimation).
+2. *Fitted distribution* fits selected distribution to the plot. Options are:
+   - [Normal](https://en.wikipedia.org/wiki/Normal_distribution)
+   - [Beta](https://en.wikipedia.org/wiki/Beta_distribution)
+   - [Gamma](https://en.wikipedia.org/wiki/Gamma_distribution)
+   - [Rayleigh](https://en.wikipedia.org/wiki/Rayleigh_distribution)
+   - [Pareto](https://en.wikipedia.org/wiki/Pareto_distribution)
+   - [Exponential](https://en.wikipedia.org/wiki/Exponential_distribution)
+   - [Kernel density](https://en.wikipedia.org/wiki/Kernel_density_estimation).
+   
+   Set *Bin width* with the slider. *Smoothing* is enabled for the Kernel density option. *Hide bars* hides bars and shows only the fitted distribution.
 3. Columns:
-
-- *Split by* displays value distributions for instances of a certain class.
-- *Stack columns* displays one column per bin, colored by proportions of class values.
-- *Show probabilities* shows probabilities of class values at selected variable.
-- *Show cumulative distribution* cumulatively stacks frequencies.
+   - *Split by* displays value distributions for instances of a certain class. Set to class target variable by default.
+   - *Stack columns* displays one column per bin, colored by proportions of class values.
+   - *Show probabilities* shows probabilities of class values at selected variable.
+   - *Show cumulative distribution* cumulatively stacks frequencies.
 
 4. If *Apply Automatically* is ticked, changes are communicated automatically. Alternatively, click *Apply*.
 
 For continuous attributes, the attribute values are also displayed as a histogram. It is possible to fit various distributions to the data, for example, a Gaussian kernel density estimation. *Hide bars* hides histogram bars and shows only distribution (old behavior of Distributions).
 
-For this example, we used the *Iris* dataset.
-
 ![](images/Distributions-Continuous.png)
 
-In class-less domains, the bars are displayed in blue. We used the *Housing* dataset.
+In class-less domains, the bars are displayed in blue. We used the *Housing* dataset for the graph below.
```

Comment:
As above, perhaps we'd avoid the term "domain" that had a (vaguely defined) meaning in 90s ML, but tells nothing to a modern data analyst. In this case, it could be "In data sets without a target variable"...